rego/rego.go

```diff
@@ -89,6 +89,7 @@ type EvalContext struct {
 	instrumentation  *topdown.Instrumentation
 	partialNamespace string
 	tracers          []topdown.Tracer
+	queryTracers     []topdown.QueryTracer
```

Comment:
Why aren't we wrapping the old topdown.Tracers so that we only have to work with the new topdown.QueryTracer (I'm assuming there's a reason for this, it just wasn't immediately obvious.) If there's no reason, I could see that being a worthwhile change to make the package more maintainable.

Reply:
Fixed in updated PR w/ refactors and perf changes.